pyrene/src/components/Paragraph/Paragraph.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FunctionComponent, ReactNode } from 'react';
+
+import './paragraph.css';
+
+/**
+* Paragraphs hold text that is not part of other components.
+*/
+export interface ParagraphProps {
+  /**
+  * The paragraph text. This can be HTML with <br/> or <b>.
+  */
+  children: ReactNode
+}
+
+const Paragraph: FunctionComponent<ParagraphProps> = ({ children }: ParagraphProps) => (
+  <div styleName="paragraph">
```

Comment:
Will this cause conflicts with the stylename/clsx changes?

Reply:
The `clsx` replacement is done already for the `pyrene` subproject. `styleName` will be done later on. So for now, it is all good. Thanks a lot for your time I appreciate a lot.